qqmlsortfilterproxymodel_plugin.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <QQmlExtensionPlugin>
+
+class QQmlsortfilterproxymodelPlugin : public QQmlExtensionPlugin
```

Comment:
@saidinesh5 Hmm... we'll see what the tests say. I've had enough problems with the stupid DLL import/export nonsense, so I'm afraid this may be no different. But we'll see.

While I'm at it, I just noticed the name of this class. I know it was generated, but could you please change this to `QQmlSortFilterProxyModelPlugin`?